src/components/Input.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import {
+  InputContainer,
+  Label,
+  Textarea,
+} from "/src/components/styles/InputStyle.js";
+
+export default function Input({ label, placeholder }) {
+  const [isFilled, setIsFilled] = useState(false);
+
+  const handleBlur = (event) => {
+    setIsFilled(event.target.value !== "");
+  };
+
+  return (
+    <InputContainer>
+      <Label>{label}</Label>
+      <Textarea
+        placeholder={placeholder}
+        onBlur={handleBlur}
+        className={isFilled ? "filled" : ""}
```

Comment:
You define a class(Name) attribute here, but I wasn't able to find an instance where you actually use/refer to this class elsewhere. If this class is necessary: please make clear why you need this/where you are currently using this class or if this is for future use. Or - in case I overlooked this - please point to where it is currently used.

Reply:
The className attribute is important for adding CSS classes dynamically to the Textarea component. In this case, it adds a "filled" class when the Textarea has content. This can be used to apply specific styles, such as changing the border color, background color, or other visual indicators to show that the Textarea is filled.

Reply:
Sure, but I still don't see what you're using it for. Anyway, go ahead, if you are happy with it.